cmd/minikube/cmd/config/profile_list.go

```diff
@@ -115,77 +113,14 @@ func profileStatus(p *config.Profile, api libmachine.API) string {
 	if len(cps) == 0 {
 		exit.Message(reason.GuestCpConfig, "No control-plane nodes found.")
 	}
-
-	status := "Unknown"
-	healthyCPs := 0
-	for _, cp := range cps {
-		machineName := config.MachineName(*p.Config, cp)
-
-		ms, err := machine.Status(api, machineName)
-		if err != nil {
-			klog.Warningf("error loading profile (will continue): machine status for %s: %v", machineName, err)
-			continue
-		}
-		if ms != state.Running.String() {
-			klog.Warningf("error loading profile (will continue): machine %s is not running: %q", machineName, ms)
-			status = ms
-			continue
-		}
-
-		host, err := machine.LoadHost(api, machineName)
-		if err != nil {
-			klog.Warningf("error loading profile (will continue): load host for %s: %v", machineName, err)
-			continue
-		}
-
-		hs, err := host.Driver.GetState()
-		if err != nil {
-			klog.Warningf("error loading profile (will continue): host state for %s: %v", machineName, err)
-			continue
-		}
-		if hs != state.Running {
-			klog.Warningf("error loading profile (will continue): host %s is not running: %q", machineName, hs)
-			status = hs.String()
-			continue
-		}
-
-		cr, err := machine.CommandRunner(host)
-		if err != nil {
-			klog.Warningf("error loading profile (will continue): command runner for %s: %v", machineName, err)
-			continue
-		}
-
-		hostname, _, port, err := driver.ControlPlaneEndpoint(p.Config, &cp, host.DriverName)
-		if err != nil {
-			klog.Warningf("error loading profile (will continue): control-plane endpoint for %s: %v", machineName, err)
-			continue
-		}
-
-		as, err := kverify.APIServerStatus(cr, hostname, port)
-		if err != nil {
-			klog.Warningf("error loading profile (will continue): apiserver status for %s: %v", machineName, err)
-			continue
-		}
-		status = as.String()
-		if as != state.Running {
-			klog.Warningf("error loading profile (will continue): apiserver %s is not running: %q", machineName, hs)
-			continue
-		}
-
-		healthyCPs++
+	statuses, err := cluster.GetStatus(api, p.Config)
```

Comment:
no need to say GetStatus, per golang best practices, it can be called cluster.Status(...)

Reply:
`cluster.Status` is a struct, so either the struct needs to be renamed, or we leave the function name as it is.